bonsai-core/src/main/scala/com/stripe/bonsai/IndexedBitSet.scala

```diff
@@ -40,6 +38,13 @@ final class IndexedBitSet(
     ((word >>> bitOffset) & 1) != 0
   }
 
+  // Returns the 10-bit block-relative rank stored in the level 2 "index".
+  // We pack in 3 ranks per 32-bit word, so getting them is a bit complicated.
+  private def getLevel2(i: Int): Int = {
```

Comment:
Yeah - I'm always a bit unsure about when to use `@inline`. I just don't have enough intuition around the size limits for JVM inlining and whatnot, so I think I'd punt this to you.